src/otx/recipe/detection/rtmdet_tiny.yaml

```diff
@@ -0,0 +1,151 @@
+model:
+  class_path: otx.algo.detection.rtmdet.RTMDetTiny
+  init_args:
+    label_info: 80
+
+    optimizer:
+      class_path: torch.optim.AdamW
+      init_args:
+        lr: 0.0007
+        weight_decay: 0.05
+
+    scheduler:
+      class_path: otx.core.schedulers.LinearWarmupSchedulerCallable
+      init_args:
+        num_warmup_steps: 3
+        main_scheduler_callable:
+          class_path: lightning.pytorch.cli.ReduceLROnPlateau
+          init_args:
+            mode: max
+            factor: 0.1
+            patience: 4
+            monitor: val/map_50
+
+engine:
+  task: DETECTION
+  device: auto
+
+callback_monitor: val/map_50
+
+data: ../_base_/data/torchvision_base.yaml
+overrides:
+  gradient_clip_val: 35.0
+  data:
+    task: DETECTION
+    config:
+      image_color_channel: BGR
+      stack_images: true
+      data_format: coco_instances
+      train_subset:
+        batch_size: 8
+        to_tv_image: False
+        transforms:
+          - class_path: otx.core.data.transform_libs.torchvision.CachedMosaic
+            init_args:
+              img_scale:
+                - 640
+                - 640
+              pad_val: 114.0
+              max_cached_images: 20
+              random_pop: false
+          - class_path: otx.core.data.transform_libs.torchvision.RandomResize
+            init_args:
+              scale:
+                - 1280
+                - 1280
+              ratio_range:
+                - 0.5
+                - 2.0
+              keep_ratio: true
```

Comment:
could you clarify the relationship between ratio_range and keep_ratio?

Reply:
`ratio_range` is to decide what `scale` will be used, and `keep_ratio` is to keep the aspect ratio when resizing the image.
For example, if `ratio_range` is 0.5, `scale` will be 640 (=1280 * 0.5). If a given image is (160, 320), the image will become (320, 640).